added mux with out any sources (#16667)
tldr-pages/tldr: pages/linux/uinput.md

# uinput

> > An interception tool writes input events to a virtual keyboard device using /dev/uinput.

Reviewer: ```suggestion
> An interception tool for writing input events to a virtual keyboard device using /dev/uinput.
```

> > An interception tool for writing input events to a virtual keyboard device using /dev/uinput.

Reviewer 2: ```suggestion
> Intercept and write input events to a virtual keyboard device using /dev/uinput.
```

> More information: <https://gitlab.com/interception/linux/tools/-/tree/master?ref_type=heads#uinput>.

- Show resulting YAML device description merge and exit (dry-run):

`uinput -p`

- Merge YAML device description(s) to resulting virtual device:

`sudo uinput -c {{path/to/device1.yaml path/to/device2.yaml ...}}`

- Merge reference device description from device node(s) to resulting virtual device:

> `sudo uinput -d {{/dev/input/eventX}}`

Reviewer 2: ```suggestion
`sudo uinput -d {{/dev/input/eventX /dev/input/eventY ...}}`
```

> `sudo uinput -d {{devnode1 devnode2 ...}}`

Reviewer 2: You said earlier that devnode is the same as `/dev/whatever`, right? If so, I would prefer the more explicit wording.

Author: yes

Author: But not all the content of the /dev/ are devnode

Reviewer 2: Then you need to make the example explicit enough so that the user won't be mistaken. Similarly to `intercept.md`.

Author: so here are the real input devices `/dev/input/eventX`
